input/pagecontent/smart-web-messaging.md

```diff
@@ -371,7 +372,7 @@ can create a list of SMART Web Messaging API calls:
 | Property              | Optionality  | Type   | Description |
 | --------------------- | ------------ | ------ | ----------- |
 | `resource`            | REQUIRED for `scratchpad.create` and `scratchpad.update`, PROHIBITED for `scratchpad.delete`  | object | Conveys resource content as per CDS Hooks Action's `payload.resource`. |
-| `location`            | REQUIRED for `scratchpad.delete` and `scratchpad.update`,  PROHIBITED for `scratchpad.create`  | string | When used for updates, the id in the `location` value SHALL match the id in the supplied resource. |
+| `location`            | REQUIRED for `scratchpad.delete` and `scratchpad.update`,  PROHIBITED for `scratchpad.create`, OPTIONAL for `scratchpad.read`  | string | When used for updates, the id in the `location` value SHALL match the id in the supplied resource. When used for a read, the returned resource SHALL match the provided value; however, if no value is provided the response may contain any subset of the scratchpad contents. |
```

Comment:
Is this supposed to take the form `{{type}}/{{id}}`? Is that the only allowed option? If so, we should state this.

Reply:
I thought it was spelled out elsewhere in the IG, but I don't see it.  Updated.